### PR TITLE
PP9-10420 - Shares: date in DP for contents in shared links looks as fluid code

### PR DESCRIPTION
**Pull request by tc4rdpp**
- Services extensions: swapped tap function call by mergeMap
 - Share detail component: refactored due to deprecation on forkJoin function call

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services.extensions.ts [hunk 1]
```diff
@@ -75,41 +75,59 @@ class ContentService extends generated.ContentService {
     public create(resolveBehaviors: ContentResolveBehavior[] | null | undefined, allowMissingDependencies: boolean | undefined,
         timeout: string | null | undefined, waitSearchDocCreation: boolean | undefined, contentCreateRequest: ContentCreateRequest): Observable<ContentDetail> {
         return this.createCore(resolveBehaviors, allowMissingDependencies, timeout, waitSearchDocCreation, contentCreateRequest).pipe(
-            tap(async content => await this.liquidRenderingService.renderNestedDisplayValues(content))
+            mergeMap(async  content => {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services.extensions.ts]: spacing

File: src/picturepark-sdk-v1-angular/projects/share-viewer/src/app/share-detail/share-detail.component.ts [hunk 2]
```diff
@@ -38,16 +39,16 @@ export class ShareDetailComponent implements OnInit {
...
-      shareDetail: this.shareService.getShareJson(searchString, null),
-      customerInfo: this.infoService.getInfo()
-    });
+    const shareInfo = forkJoin([
+       this.shareService.getShareJson(searchString, null),
+       this.infoService.getInfo()
+    ]);
 
     shareInfo.subscribe({
       next: (values) => {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/share-viewer/src/app/share-detail/share-detail.component.ts]: Use array destructuring

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services.extensions.ts (hunk 1 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services.extensions.ts]: Fixed

File: src/picturepark-sdk-v1-angular/projects/share-viewer/src/app/share-detail/share-detail.component.ts (hunk 2 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/share-viewer/src/app/share-detail/share-detail.component.ts]: Done

**Pull request merged**